src/prefix.rs

```diff
@@ -49,6 +51,20 @@ where
     de_fn_v: DeserializeVFn<T>,
 }
 
+impl<K, T> Debug for Prefix<K, T>
+where
+    K: KeyDeserialize,
+    T: Serialize + DeserializeOwned,
+{
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Prefix")
+            .field("storage_prefix", &self.storage_prefix)
+            .field("data", &self.data)
```

Comment:
Seems like the `PhantomData` is has a different debug implementation for Rust 1.64.0 and 1.67.1. In the CI we get `"Prefix { storage_prefix: [0, 3, 108, 111, 108, 0, 1, 8], data: PhantomData, pk_name: [], .. }"`.

Is the field relevant for debugging? Or should we just omit it?

Reply:
idk what the end-user would need PhantomData for. As I understood it, PhantomData is just a placeholder for some arbitrary data, or something like that. If it turns out to be important for the end-user, then we can always adjust it in a future PR